tests/test_hookman_generator/generate_plugin.c

```diff
@@ -1,6 +1,5 @@
 #include "hook_specs.h"
 
-INIT_HOOKS()
 
 // HOOK_FRICTION_FACTOR(v1, v2){}
-// HOOK_FRICTION_FACTOR_2(v1, v2){}
+// HOOK_FRICTION_FACTOR_2(v1, v2){}
```

Comment:
"No newline at end of file" is that a problem?

Reply:
https://stackoverflow.com/questions/72271/no-newline-at-end-of-file-compiler-warning

Reply:
https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206

>  **3.206 Line**
> A sequence of zero or more non- <newline> characters plus a terminating <newline> character.

Reply:
Commit 7e57049 fix this issue ;)